smash/factory/net/net.py

```diff
@@ -514,13 +514,22 @@ def _fit_d2p(
             random_state=random_state,
         )
 
+        # % Initialize optimizer for the parameterization NN if used
+        ind = PY_OPTIMIZER.index(optimizer.lower())
```

Comment:
I have not looked at the PY_OPTIMIZER yet; well well and so quickly done yet  ;)